### PR TITLE
HdrHistogram thirdparty lib

### PR DESCRIPTION
**Pull request by toly-kournik**
A part of an effort of making concord-bft library being built in a standard way on multiple platforms.
This will make it more development friendly and attractive to users outside vmware.
Add an option to compile a HdfHistogram thirdparty library as a part of a build using ExternalProject_Add().

Docker build stays intact!

### CODE REVIEW

File: logging/include/Logging.hpp [hunk 1]
```diff
@@ -67,7 +67,7 @@ class LoggerImpl {
 
     // clang-format off
     std::cout << std::put_time(std::localtime(&cur_time.tv_sec), "%FT%T.") << (int)cur_time.tv_usec / 1000
-       << "|" << LoggerImpl::LEVELS_STRINGS[l].c_str()
+       << "|" << LoggerImpl::LEVELS_STRINGS[l]
```
Comment (f-squirrel) [inline logging/include/Logging.hpp:70]: Is it related to the HDR historgram?

**Review by f-squirrel — CHANGES_REQUESTED** [review]
Did we reach a consensus on this approach?

**Review by yontyon — APPROVED** [review]
LGTM!

**Comment by robem** [comment]
@toly-kournik Would you mind providing a commit message which explains what this PR is changing and why?

File: logging/include/Logging.hpp (hunk 1 above)
Comment (toly-kournik) [reply logging/include/Logging.hpp:70]: This is related to a compilation on MacOS with a most recent clang compiler (12.05)

**Pull request merged**